### ISSUE TITLE
Single selections with ajax don't set aria-label unless a placeholder is set

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/selectWoo#7]
When a single selection select2 is powered by ajax, unless a `data-placeholder` is used the `aria-label` is not set.

This can be replicated in the playground file. If you remove the `data-placeholder` from the select, no `aria-label` is set on the select2, resulting in various a11y tool failures.
